app/temporal/src/workflows/importer.workflow.ts

```diff
@@ -251,8 +261,26 @@ export async function importer(params: ImporterWorkflowParams) {
       },
     }
   );
+  setHandler(columnValidationUpdate, (params) => {
```

Comment:
Müsste man da eigentlich nochmal eine Revalidation anstoßen?

Reply:
Ne die revalidation wird über das FrontEnd durch das SetValue getriggered.

Reply:
Was passiert, wenn ich in der Aufrufenden Plattform eine neue Abteilung anlege (weil eine vergessen wurde) und ich dann einen Patch an DD sende? Dann würden die Daten ja nicht neu validiert, oder?

Reply:
@Jank1310 du kannst es gerne nochmal testen, aber bei mir hat das ohne problem funktioniert.
![image](https://github.com/Jank1310/datadolphin/assets/12624982/d3e97e9e-909a-4587-8d8c-0e831023d377)

![image](https://github.com/Jank1310/datadolphin/assets/12624982/94043f1d-7766-4b52-9dd6-2baee6e5e2fb)